core/domain/question_services_test.py

```diff
@@ -116,5 +119,59 @@ def test_update_question(self):
         model = question_models.QuestionModel.get(question_id)
         self.assertEqual(model.question_data, new_question_data)
         self.assertEqual(
-            model.question_data_schema_version, question_data_schema_version)
-        self.assertEqual(model.language_code, language_code)
+            model.question_data_schema_version,
+            self.question_data_schema_version)
+        self.assertEqual(
+            model.language_code, self.language_code)
+
+    def test_compute_summary_of_question(self):
+        question_summary = question_services.compute_summary_of_question(
+            self.question, self.owner_id, self.status)
+
+        self.assertEqual(question_summary.id, 'dummy')
+        self.assertEqual(
+            question_summary.question_data,
+            html_cleaner.clean(''))
+        self.assertEqual(question_summary.language_code, 'en')
+        self.assertEqual(question_summary.status, 'private')
+
+    def test_get_question_summaries_by_creator_id(self):
+        question_services.add_question(self.owner_id, self.question)
+        question_summaries = (
+            question_services.get_question_summaries_by_creator_id(
+                self.owner_id))
+
+        for question_summary in question_summaries:
+            self.assertEqual(question_summary.id, 'dummy')
+            self.assertEqual(
+                question_summary.question_html_data, '')
+            self.assertEqual(question_summary.language_code, 'en')
+            self.assertEqual(question_summary.status, 'private')
+
+    def test_admin_can_publish_and_edit_question(self):
```

Comment:
Did you add tests for who can edit the question before  publishing/approval? Only the creator should be able to.

Reply:
added